### PR TITLE
Show fuel as number of turns instead of mps

### PR DESCRIPTION
**Pull request by psampathkumar**
It is as the title suggests. Let me know if fuel should be a separate line or in the same line as moves. It is currently in the same line.

### CODE REVIEW

File: common/unit.c [hunk 1]
```diff
@@ -1328,15 +1328,17 @@ void unit_activity_astr(const struct unit *punit, struct astring *astr)
...
   case ACTIVITY_IDLE:
     if (utype_fuel(unit_type_get(punit))) {
-      int rate, f;
+      int f;
 
-      rate = unit_type_get(punit)->move_rate;
+      /* Rate has been commented out because showing fuel as number of turns 
+         doesnt need rate */
+      // int rate;
+      // rate = unit_type_get(punit)->move_rate;
```
Comment (lmoureaux) [inline common/unit.c]: Let's just remove it. Git history can bring it back if needed.

File: common/unit.c [hunk 2]
```diff
@@ -1328,15 +1328,17 @@ void unit_activity_astr(const struct unit *punit, struct astring *astr)
...
       f = ((punit->fuel) - 1);
 
       /* Add in two parts as move_points_text() returns ptr to static
        * End result: "Moves: (fuel)moves_left" */
-      astr_add_line(astr, "%s: (%s)", _("Moves"),
-                    move_points_text((rate * f) + punit->moves_left, FALSE));
+      astr_add_line(astr, "%s: (Fuel: %d T)", _("Moves"), f);
```
Comment (lmoureaux) [inline common/unit.c]: This should be translated: `_("%s: (Fuel: %d T)")`

**Review by lmoureaux — COMMENTED** [review]
> Let me know if fuel should be a separate line or in the same line as moves. It is currently in the same line.

I think there's enough horizontal space. The total number of tiles can be wrong so in can be removed.

**Comment by psampathkumar** [comment]
> The total number of tiles can be wrong so in can be removed.

I don't understand. What can be removed ?

File: common/unit.c [hunk 3]
```diff
@@ -1330,15 +1330,11 @@ void unit_activity_astr(const struct unit *punit, struct astring *astr)
     if (utype_fuel(unit_type_get(punit))) {
       int f;
 
-      /* Rate has been commented out because showing fuel as number of turns 
-         doesnt need rate */
-      // int rate;
-      // rate = unit_type_get(punit)->move_rate;
```
Comment (lmoureaux) [inline common/unit.c]: This :)

Comment (psampathkumar) [reply common/unit.c]: Oh alright. Removed that already.

**Comment by cgalik** [comment]
Nice work, @pranavtifr!

**Pull request merged**